tests/system/lab/test

```diff
@@ -863,14 +862,26 @@ event_toggled_sent_clear
 
 echo "======================= TOGGLE EVENT NTFSN and NTFCL TEST COMPLETE ======================="
 
+if [[  ${VERIFY_EGRESS_TO_PI} == 1 ]]
+  then
+     setup_north_pi_egress
+     curl -X PUT  http://localhost:8081/fledge/schedule/disable -d '{"schedule_name": "PI Server"}'
```

Comment:
Use `$URL` instead of hardcoded base_uri `http://localhost:8081/`
Also add comment/note for disabling of a schedule.